src/components/TimerDisplay.js

```diff
@@ -0,0 +1,11 @@
+import React, { useState } from 'react';
+
+const TimerDisplay = (props) =>{   
+    return(
+        <div>
+            Time spent: {props.timeElapsed}
+        </div>
+    )
+}
+
+export default TimerDisplay;
```

Comment:
https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad
https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/
https://www.reddit.com/r/webdev/comments/7j9nvy/why_we_have_banned_default_exports_in_javascript/

W skrócie, od wszystkiego są wyjąki, jednak w większości przypadków `default` export będzie stwarzał problemy tj. gorszy DX czy ręczna aktualizacja plików `index.ts`/`index.js`. Osobiście kieruję się zasadą korzystania wszędzie tam gdzie mogę z `named` exportów. Natomiast `default` exportów używam tam gdzie wymaga tego ficzer/konwencja, np. przy `lazy` loadingu.

Reply:
Jak w takim razie eksportować na przykładzie `TimerDisplay`?

Reply:
Przy deklaracji komponentu dopisz `export` i to wystarczy, potem w `index.ts/js` używasz takiej konwencji: `export * from './TimerDisplay'. Żeby nie utrzymywać indexów samodzielnie możesz użyć `https://www.npmjs.com/package/create-ts-index` i `https://github.com/paulmillr/chokidar`.